addons/website/static/src/js/menu/edit.js

```diff
@@ -247,6 +247,11 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             });
         });
 
+        // We hide the cookie bar because contenteditable="true" would force the bar to stay visible while initialization of edit mode.
+        const $cookieBar = $('#website_cookies_bar');
+        if ($cookieBar.length) {
+            $cookieBar.hide();
+        }
```

Comment:
Yes but not, the fix is not right:
- First, why is there that contenteditable on that element now and not before ? Didn't we already have the new editor in 14.0? No bug there ? Is this property added because it is positioned somewhere special in the page ?
Did you investigate anything and have useful info to provide or did you directly end up with the specific fix ?
- If we really need to keep that contenteditable, then make a generic fix somewhere around `_updateInvisibleDOM` stack, instead of this cookie specific fix, otherwise any other `o_snippet_invisible` element that would receive a contenteditable=true will have the same bug
You can have a look and drag&drop a popup snippet, add it a contenteditable=true and you will have the same issue.